custom_components/enphase_envoy_custom/const.py

```diff
@@ -25,6 +25,27 @@
 CONF_USE_ENLIGHTEN = "use_enlighten"
 
 SENSORS = (
+    SensorEntityDescription(
+        key="grid_import_power",
+        name="Current Grid Import Power",
+        native_unit_of_measurement=POWER_WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.POWER,
+    ),
+    SensorEntityDescription(
+        key="grid_export_power",
+        name="Current Grid Export Power",
+        native_unit_of_measurement=POWER_WATT,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.POWER,
+    ),
+    SensorEntityDescription(
+        key="daily_net_import_energy",
+        name="Today's Net Import Energy",
+        native_unit_of_measurement=ENERGY_WATT_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
I don't believe `MEASUREMENT` is correct here. This should be `TOTAL` and you will also need to set `last_reset` properly.

https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics